cli/index.js

```diff
@@ -14,22 +14,37 @@ const yargs = require('yargs'),
   pkg = require('../package.json'),
   notifier = updateNotifier({
     pkg
-  });
+  }),
+  commands = {
+    cfg: 'config',
+    l: 'lint',
+    i: 'import',
+    e: 'export',
+    c: 'compile'
+  },
+  listCommands = Object.keys(commands).concat(Object.values(commands));
 
 if (notifier.update) {
   // note: this will only check for updates once per day
   notifier.notify();
   process.exit(0);
 }
 
+let thisCommand = process.argv.filter(arg => listCommands.includes(arg))[0];
+
+if (!thisCommand) {
+  console.error('invalid command');
+  process.exit();
+}
+
```

Comment:
can you add a comment here saying how this allows both shorthand and full commands to work?

```
// this allows for shorthand (cfg, l, i, e, c) to work as well as full command (config, lint, import, export, compile
```